projects/demo/src/modules/customization/viewport/viewport.component.ts

```diff
@@ -1,13 +1,29 @@
 import {Component} from '@angular/core';
 import {changeDetection} from '@demo/emulate/change-detection';
-import {type TuiDocExample} from '@taiga-ui/addon-doc';
+import {
+    TuiDocCodeModule,
+    type TuiDocExample,
+    TuiDocExampleModule,
+    TuiDocPageModule,
+} from '@taiga-ui/addon-doc';
+
+import {TuiViewportExample1} from './examples/1';
+import {TuiViewportExample2} from './examples/2';
 
 @Component({
+    standalone: true,
     selector: 'example-tui-viewport',
+    imports: [
+        TuiDocPageModule,
+        TuiDocExampleModule,
+        TuiViewportExample1,
+        TuiViewportExample2,
+        TuiDocCodeModule,
```

Comment:
`TuiAddonDocModule` instead of `TuiDocPageModule` & `TuiDocExampleModule` & `TuiDocCodeModule` ?

Reply:
fixed